ssha/config.py

```diff
@@ -172,6 +205,10 @@ def load(name):
     # first identity file in the SSH config that has a matching ".pub" file.
     # The SSM document command would then add this public key to a user account
     # so that SSH key-based authentication will work.
+
+    if is_used_as_variable('ssh.temporary_key'):
```

Comment:
This is breaking up the comment above which is for the code below. The comment no longer makes sense. Please move this code and give it its own comment.